src/manual/tasks/artifacts.md

```diff
@@ -0,0 +1,45 @@
+---
+filename: tasks/artifacts.md
+title: Artifacts
+order: 35
+---
+
+But by far the most commonly-used data from a run are its artifacts. These are
+HTTP entities containing output data from the task execution.
+
+Unlike most API methods which return a JSON body, requesting an artifact from
+the Queue service returns the artifact itself, possibly via one or more HTTP
+redirects. This means that -- at least for public artifacts which require no
+authentication -- any sufficiently robust HTTP client can download an artifact
+directly.
+
+**NOTE**: Not all clients are "sufficiently robust"! The artifact interface
+makes heavy use of redirects, and artifacts may make use of other web-standard
+features such as content encoding.  Like any distributed system, requests may
+fail, too, and a robust client should retry. Out of the box, `curl` and `wget`
+do not handle most of these cases.
```

Comment:
@jhford did I miss a note about that being finished? I thought it was still in development -- but when it's done, yes, definitely!